framework/src/Volo.Abp.IdentityModel/Volo/Abp/IdentityModel/IdentityModelAuthenticationService.cs

```diff
@@ -76,21 +81,36 @@ public virtual async Task<string> GetAccessTokenAsync(IdentityClientConfiguratio
                 throw new AbpException($"Could not retrieve the OpenId Connect discovery document! ErrorType: {discoveryResponse.ErrorType}. Error: {discoveryResponse.Error}");
             }
 
-            var tokenResponse = await GetTokenResponse(discoveryResponse, configuration);
-
-            if (tokenResponse.IsError)
+            var cacheKey = CalculateCacheKey(discoveryResponse, configuration);
```

Comment:
Why not including the `GetDiscoveryResponse` call to the cached part? Should we always call it?

Reply:
Sorry I forgot it, I will cache it.